cluster/addons/addon-manager/Makefile

```diff
@@ -15,24 +15,10 @@
 IMAGE=staging-k8s.gcr.io/kube-addon-manager
 ARCH?=amd64
 TEMP_DIR:=$(shell mktemp -d)
-VERSION=v8.7
+VERSION=v8.8
```

Comment:
is there any significance to the major/minor of this version?

Reply:
Not really, I should probably call it `v8.7-r1`?